app/assets/stylesheets/application.scss

```diff
@@ -127,7 +127,7 @@ section h2 {
 @media (min-width: 992px) {
   #mainNav {
     border-color: transparent;
-    background-color: transparent;
+    background-color: #4e4a4aa6;
```

Comment:
Looks much better but is still semi transparent. Do we want to make it completely solid so there are no conflicts with text in the nav and text in the page?

Reply:
@muydanny can decide, he wasn't a fan of the stronger opacity I had tried behind the podcast description.

Reply:
okay, sounds good to keep it as is for now then.